### PR TITLE
Rework CodeCov integration:

### LINKED ISSUE #1480: Investigate Codecov applicability

### PR DESCRIPTION
**Pull request by ericbottard**
Remove cobertura and reuse our existing jacoco setup

Fixes #1480

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -136,19 +136,41 @@
...
+					</excludes>
+					<!-- see http://stackoverflow.com/questions/18107375/getting-skipping-jacoco-execution-due-to-missing-execution-data-file-upon-exec -->
+					<argLine>${argLine}</argLine>
 				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.jacoco</groupId>
+				<artifactId>jacoco-maven-plugin</artifactId>
+				<!--<configuration>-->
+					<!--<destFile>${sonar.jacoco.reportPath}</destFile>-->
```
Comment (ilayaperumalg) [inline pom.xml]: Why is this commented out now? wondering what happens if in the Sonar CI environment without this.

Comment (ericbottard) [inline pom.xml]: Sorry, was meant to be *removed*.
From the Slack discussion with @ghillert , turns out we're not using Sonar...

Comment (ilayaperumalg) [inline pom.xml]: ok, will remove while merging.

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Comment by ilayaperumalg** [comment]
Merged as 876db9831226ba8239c0d85703d4d40587ed3038 with the removal of the above commented lines.

**Pull request closed without merging**